app/serializers/school_serializer.rb

```diff
@@ -0,0 +1,6 @@
+# frozen_string_literal: true
+
+class SchoolSerializer < Blueprinter::Base
```

Comment:
It would be good to add unit tests for these serializers as well though this one doesn't have much logic yet.

Reply:
I do test it indirectly spec/requests/api/school_search_spec.rb - so I could extend that to also have name attribute - or would you prefer pure serializer tests

Reply:
@amazimbe

Reply:
Yes, I'd prefer pure serializer tests as well. Not so much for this one but to declare the intention that we test exhaustively 😉 . If we don't add them now probably the next person won't too.

Reply:
Yeah - you do have a point there - plus since we are using a new serializer we need to know the patterns for adding more

Reply:
Done